.env

```diff
@@ -0,0 +1,9 @@
+MYSQL_USER=root
```

Comment:
MYSQL_USER is reserved variable name in MySQL, so whether specify other name, or specify other user, because root is created by default.